From 532116d4dd16632806ed4a036c2e89b966221031 Mon Sep 17 00:00:00 2001
From: Bill Pemberton <wfp5p@virginia.edu>
Date: Wed, 10 Oct 2012 16:54:01 -0400
Subject: [PATCH 05/11] mount: add -c abbreviation for --no-canonicalize to
 man page

The --no-canonicalize option can also be set using -c.  The --help for
mount shows this option but the man page did not.  Add -c to the man
page.

Signed-off-by: Bill Pemberton <wfp5p@virginia.edu>
---
 sys-utils/mount.8 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sys-utils/mount.8 b/sys-utils/mount.8
index dce4b8c..83a4518 100644
--- a/sys-utils/mount.8
+++ b/sys-utils/mount.8
@@ -552,7 +552,7 @@ Mount without writing in
 This is necessary for example when
 .I /etc
 is on a read-only filesystem.
-.IP "\fB\-\-no\-canonicalize\fP"
+.IP "\fB\-c, \-\-no\-canonicalize\fP"
 Don't canonicalize paths. The mount command canonicalizes all paths 
 (from command line or fstab) and stores canonicalized paths to the 
 .IR /etc/mtab
-- 
1.7.11.7

